CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,2 @@
+- Use device apikey if exists in getEffectiveApiKey for command handling
+- Upgrade NodeJS version from 8.16.1 to 10.17.0 in Dockerfile due to Node 8 End-of-Life 
```

Comment:
I think some similar to https://github.com/telefonicaid/iotagent-ul/pull/398 has happened here.

Reply:
Fixed 7d0a33a